templates/script/script.js

```diff
@@ -1,153 +1,133 @@
-    if(window.innerWidth < 768)
-    {
-        document.getElementById('form').style.width="100%";
-        document.getElementById('table_div').style.width="100%";
-        document.getElementById('name').style.width="100%";
-        document.getElementById('phone').style.width="100%";
-        document.getElementById('email').style.width="100%";
-    }
+var elEmail = document.getElementById("email");
+var elName = document.getElementById("name");
+var elPhone = document.getElementById("phone");
+var btPost = document.getElementById('post');
+var btGet = document.getElementById('get');
+var btClear = document.getElementById('clear');
+var divTable = document.getElementById('table_div');
+var fForm = document.getElementById('form');
 
+ResizeElements();
```

Comment:
1. This and the above variables are in the window object. This is a pollution of global object.
2. Generally, the logic is messed up. Please, think of splitting it logically onto different files